pkg/landscaper/controllers/installations/controller.go

```diff
@@ -480,3 +498,26 @@ func (c *Controller) setInstallationPhaseAndUpdate(ctx context.Context, inst *ls
 
 	return lsError
 }
+
+func (c *Controller) checkForDuplicateExports(ctx context.Context, inst *lsv1alpha1.Installation) error {
```

Comment:
With the new reconcile logic this works well for subinstallations but with respect to root installations I see problems with race conditions, e.g. if root inst RI-A is just creating its export data including export data EX and during that its spec is modified such that the new spec does not export EX anymore. This does not stop the export data generation of RI-A. If now another root inst RI-B is created also exporting EX and it checks the exports of RI-A is only sees the new spec without EX but EX is currently created resulting in potential problems. Or do I miss here something? 

Perhaps another strategy is to fail working on an installation if it tries to create some export data and this already exists and was created by another installation. Then we do not need a global check and would have no race conditions.

Reply:
You are right, this could happen, although it is somewhat unlikely.
I will add an additional check to the export creation. While only that check would be enough, I don't like the idea of detecting this problem during export creation, so I will keep the other checks in.

Reply:
ok